docs/content/docs/install/github_webhook.md

```diff
@@ -13,24 +13,57 @@ API](https://docs.github.com/en/rest/guides/getting-started-with-the-checks-api)
 therefore the status of
 the tasks will be added as a Comment on the PullRequest and not through the **Checks** Tab.
 
-After you have finished the [installation](/docs/install/installation), you need to create
-a GitHub personal access token for Pipelines-as-Code GitHub API operations.
+gitops comment (ie: /retest /ok-to-test) with GitHub webhook is
+not supported. If you need to restart the CI you will need to generate a new
+commit. You can make it quick with this command line snippet (adjust branchname to the name of
+the branch) :
+
+```console
+git amend -a --no-edit && git push --force-with-lease origin branchname
```

Comment:
can you please remind me why don't we support /retest on github webhook 🤔

Reply:
I can't remember exactly but I could not figure all the information needed in the payload to know which repository the retest is targetting.. feel free to have another crack at it, I spent quite a bit of time on it but I may  have missed something..